doc/active_docs/account_management_api.json

```diff
@@ -2240,7 +2240,7 @@
             "name": "id",
             "in": "path",
             "description": "ID of the user.",
-            "x-data-threescale-name": "user_ids",
+            "x-data-threescale-name": "provider_users_ids",
```

Comment:
In `sumary` we have "Gets the user of the provider account by ID." Do we need the change here?

Reply:
There is a change from `user_ids` to `provider_users_ids` in this PR. My question is whether we and why we need this change.

Reply:
Ah... well, this is fixing a bug described in the PR description :)

`user_ids` list the users of the **buyers**, `provider_users_ids` (which is a new method added by this PR) lists the users of the **provider**.

So, the summary of the endpoint is correct, but the auto-complete was not.